src/openrct2-ui/windows/TitleCommandEditor.cpp

```diff
@@ -36,16 +36,16 @@ struct TITLE_COMMAND_ORDER {
 };
 
 static TITLE_COMMAND_ORDER _window_title_command_editor_orders[] = {
-    { TITLE_SCRIPT_LOAD,        STR_TITLE_EDITOR_ACTION_LOAD_SAVE, STR_TITLE_EDITOR_ARGUMENT_SAVEFILE },
```

Comment:
Change the type on line 33. Remove the comment on line 32.

Reply:
Thanks for the feedback. I assume you mean change the `int` to `uint8_t?`

Reply:
I think I see what you mean. Change the type for `command` from 'uint16_t' to `uint8_t`?

Reply:
The struct `TitleCommand` has a member variable `uint16_t Milliseconds`. Should that be adjusted too?
[TitleSequence.h]("src/openrct2/title/TitleSequence.h")

Reply:
Command should be type of TitleScript